dataprofiler/profilers/utils.py

```diff
@@ -87,9 +89,10 @@ def shuffle_in_chunks(data_length, chunk_size):
     if not data_length or data_length == 0 \
        or not chunk_size or chunk_size == 0:
         return []
-    
-    rng = np.random.default_rng()
-    if 'DATAPROFILER_SEED' in os.environ:
+
+    rng = np.random.default_rng(settings._seed)
```

Comment:
Did we want to make this so that there is only want rng call instead of potentially 2?